moveit_ros/moveit_servo/src/servo_calcs.cpp

```diff
@@ -319,17 +303,86 @@ void ServoCalcs::stop()
   }
 }
 
+void ServoCalcs::validateParams()
+{
+  if (servo_params_.hard_stop_singularity_threshold <= servo_params_.lower_singularity_threshold)
+  {
+    RCLCPP_ERROR(LOGGER, "Parameter 'hard_stop_singularity_threshold' "
+                         "should be greater than 'lower_singularity_threshold.' "
+                         "Check yaml file.");
```

Comment:
```suggestion
                         "Check the parameters YAML file used to launch this node.");
```

Same with the others.